### DISCUSSION TITLE
[功能]: 增加星标、标签功能，通过给对话内容加星标、标签，方便后续查询历史对话

### DISCUSSION THREAD

**Discussion by vimbackground** [CherryHQ/cherry-studio#2405] [Ideas]
### 您的功能建议是否与某个问题相关?

有些话题对话下来质量比较高，后续直接查询使用即可，不用重新对话。现在只能通过搜索查内容，不能保证准确定位到。

如果有标签、星标功能，可用性会更强。

目前只能用“移动到XX助手”的方式替代，不太合适的用法。

### 请描述您希望实现的解决方案

希望能增加话题标签、或者话题星标的功能，对优质对话进行标记，方便后续查询。

- 最简单实现：直接添加星标（收藏），左侧增加一个按钮入口。需要的时候在这里翻、或者搜索历史对话
- 进阶场景：对加星标的话题，再增加一个标签功能，点击标签后进行进一步的过滤选择。
- 再进一阶：可以对筛选出来的对话，多选，然后一起导出；选择大模型自动给对话加候选标签，用户选择确认。

以上，星标类似知乎直答的收藏，标签类似perplexity的空间、秘塔搜索的专题。

**Comment by kangfenmao** [comment]
之前版本移动话题之后就不能定位了。后面我加一个收藏功能

**Comment by crybest** [comment]
特别想要这个功能

**Comment by gaoxiaobei** [comment]
同求星标功能

**Comment by EurFelux** [comment]
#7908
